01/userinfamous/wordvalue.py

```diff
@@ -0,0 +1,33 @@
+from data import LETTER_SCORES, DICTIONARY 
+import os
+
+#File path for Dictionary
+fname = os.path.join('../',DICTIONARY)
+
+#Wordlist from dictionary
+def load_words() -> list:
+    with open(fname,'r') as f:
+        wordlist = f.read().split()
+        return wordlist
```

Comment:
return should be outside the with context. The lock is lifted, even for `return` inside a with block, but its good practice to not do this: https://stackoverflow.com/questions/9885217/in-python-if-i-return-inside-a-with-block-will-the-file-still-close